HW02_B_Mimic_starter.Rmd

```diff
@@ -64,11 +90,26 @@ corvette <- mpg[mpg$model == "corvette",]
 #install
 require("ggrepel") #useful for making text annotations better, hint hint
 set.seed(42)
+
 ```
 
 Now using the mpg dataset and the corvette dataset, make this graph:
 
-```{r graoh 3 code}

```

Comment:
The figure looks great. Just as requested. I found the code for this figure a little hard to follow, but it works well! Particularly, I was confused about the "label" argument within "ggplot", perhaps adding it within "geom_text_repel" would be a bit more clear (this is the function that actually add the label, but the labels themselves are specified outside). This is totally personal though (I might be even saying something wrong too!), as it works perfectly. 

The strategy for coloring the corvette points in a different color (blue) was very clever. I just added blue points on top (for corvettes) of the black ones (for all cars), which is not a great solution. Other thing I learned!